README.md

```diff
@@ -1,12 +1,18 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/AcademySoftwareFoundation/rez/blob/main/LICENSE)<br>
 [![Release](https://shields.io/github/v/release/AcademySoftwareFoundation/rez)](https://github.com/AcademySoftwareFoundation/rez/releases)
-[![Pypy Release](https://shields.io/pypi/v/rez)](https://pypi.org/project/rez)<br>
+[![Pypi Release](https://shields.io/pypi/v/rez)](https://pypi.org/project/rez)<br>
+[![Google Group](https://img.shields.io/badge/rez--config_%28retired%29-Google%20Group-blue?style=flat-square&logo=google)](https://groups.google.com/forum/#!forum/rez-config)
+[![LFX ASWF Email List](https://img.shields.io/badge/rez--discussion-lists.aswf.io-003366?style=flat-square&logo=linuxfoundation)](https://lists.aswf.io/g/rez-discussion)
```

Comment:
```suggestion
[![Mailing List](https://img.shields.io/badge/rez--discussion-lists.aswf.io-003366?style=flat-square&logo=linuxfoundation)](https://lists.aswf.io/g/rez-discussion)
```
Since not everybody knows what LFX means, I think it's better to just say "Mailing list".